bool request
---
EEpos left_ee_pos
EEpos right_ee_pos
EEpos head_ee_pos
